topo_processor/metadata/loaders/metadata_loader_imagery_historic.py

```diff
@@ -19,37 +22,39 @@ def is_applicable(self, item: Item) -> bool:
         return True
 
     def add_metadata(self, item: Item) -> None:
-        file_name = os.path.splitext(os.path.basename(item.path))[0]
+        if not self.is_init:
+            self.read_csv()
+
+        item_path_basename = os.path.splitext(os.path.basename(item.path))[0]
+        item_dict = self.csv_dict[item_path_basename]
```

Comment:
`item_dict` may be null if the file does not exist.

Reply:
fixed
```
        if item_path_basename not in self.csv_dict:
            raise Exception(f"{item_path_basename} cannot be found in the csv.")
```